README.md

```diff
@@ -27,4 +27,53 @@ The [code](https://github.com/Microsoft/vsts-urbancode-deploy) is open sourced u
 1. npm update
 1. gulp
 
-The vsix package will be produced in `_package`, and it can be uploaded to Visual Studio Team Services Marketplace for sharing.
\ No newline at end of file
+The vsix package will be produced in `_package`, and it can be uploaded to Visual Studio Team Services Marketplace for sharing.
+
+## FAQ
+### What if I need to work with Sonatype Nexus 3.x?
+Now, the best way to upload artifacts into Nexus 3.x is to use the Maven publish capabilities.
+The quickest way to deploy a file with Maven is described [here](https://maven.apache.org/plugins/maven-deploy-plugin/usage.html).
+You basically tell Maven which file to deploy and define the repository you are deploying to in the pom.xml & settings.xml files. 
```

Comment:
use the full URL instead of "here" for display name as well to highlight the link and to indicate it is what is recommended by the plugin.